wqreports/core/pdfreport.py

```diff
@@ -37,6 +123,12 @@ class PdfReport(object):
         right-censored (non-detect). Any value in ``qualcol`` that is
         *not* in this list will be assumed to denote an uncensored
         (detected value).
+    bsIter : int (default = 10000)
```

Comment:
made these properties of the report so that they can be passed to all of the locations